README.md

```diff
@@ -3,11 +3,11 @@
 [![Build Status](https://ci.digitalbazaar.com/buildStatus/icon?job=bedrock-ledger-storage-mongodb)](https://ci.digitalbazaar.com/job/bedrock-ledger-storage-mongodb)
 
 A MongoDB ledger storage subsystem for bedrock-ledger that enables the
-storage and retrieval of ledgers, blocks, and events. The relationship
-of these objects are shown below:
+storage and retrieval of ledgers, blocks, events, and operations. The
+relationship of these objects are shown below:
 
-<img alt="Ledgers contain blocks, blocks contain events"
-  src="https://w3c.github.io/web-ledger/diagrams/blocks.svg"
+<img alt= 'Ledgers contain blocks, blocks contain events '
+  src= 'https://w3c.github.io/web-ledger/diagrams/blocks.svg '
```

Comment:
Looks like over zealous quote search & replace.

Reply:
It's probably in the multihash branch too... I'll just fix it here.